feat: move subscribe link to the calendar header

jdwillemse opened the pull request:
I've changed the styling and position of the calendar subscribe link to be inline with the page heading for the following reasons:
- The link is styled like a link because it leads to a page. Button styling would imply that it does something on the current page in our site design language
- The link pertains to the entire calendar and not a specific month so putting it at the top level creates the correct association
-  Adding the link to every month heading gives it more importance that is needed. This link will likely be used by very few people.
- The small text is not sufficiently legible.

pantierra reviewed (approved):
Makes a lot of sense. Thanks!

Commit:
feat: move subscribe link to the calendar header

gueldenstone committed:
fix calendar subscription links (#81)

SThomasP committed:
Update map to reflect space changes (#82)

Commit:
Update matrix well-known

rrrnld committed:
fix calendar subscription url (#83)

Co-authored-by: near <noreply@null.arnes.space>

alinnow committed:
fix invalid date of support page (#84)

alinnow committed:
chore: de-duplicate docker tag reference in woodpecker (#85)

alinnow committed:
add link checker to CI

alinnow committed:
make link checking the last step

alinnow committed:
chore: update hugo to v0.144

alinnow committed:
fix incorrect menu URL generation

Hugo is quite unintuitive with URL generation; a slash prefix is _not_
recommended

https://gohugo.io/functions/urls/abslangurl/#input-does-not-begin-with-a-slash

alinnow committed:
fix some blog article links

alinnow committed:
feat: generate pages for each calendar entry

- add link to title inside event modal dialog
- ctrl-/shift-/middle-click event to open page straight away
- store the fetched calendar in the hugo store
- pipe content to markdown

Closes: https://github.com/xHain-hackspace/xhain-website/issues/79

alinnow committed:
refactor: use html5 <dialog> instead of custom code

alinnow committed:
feat: render markdown in event descriptions

alinnow committed:
simplify event display CSS

alinnow committed:
emit javascript as separate files

alinnow committed:
use javascript-specific minification function

alinnow committed:
limit loading of map/calendar js to respective pages

alinnow committed:
extract map portion of main JS to own file

gueldenstone committed:
Adds healthcheck to monitor cron based CI runs (#90)

Adds an additional step to the woodpecker pipeline to monitor the cron job using healthchecks.io

gueldenstone committed:
Adds GuB blog aricle (#92)

* Adds GuB blog aricle

davedarko committed:
Update support.md (#93)

* Update support.md

changed account name to match the Empfaengerueberpruefung

* Update support.md german

davedarko committed:
Update support.md (#94)

* Update support.md

* Update support.md eng

gueldenstone committed:
Switch to GitHub CI (#96)

Autoscaler for woodpecker CI broke down. This keeps the website updated for now.

gueldenstone committed:
Fix github workflow

* deploy only on main
* use different minute of the hour than 0 to avoid congestion

gueldenstone committed:
every 5 minutes for testing

SuperQ committed:
Pin GitHub actions

It's recommended to pin GitHub Actions to their git SHA to improve
supply chain security.
* Enable dependabot to update actions monthly.

Signed-off-by: SuperQ <superq@gmail.com>

gueldenstone committed:
add manual dispatch for debugging.

dependabot[bot] committed:
Bump easingthemes/ssh-deploy from 5.1.0 to 5.1.1

Bumps [easingthemes/ssh-deploy](https://github.com/easingthemes/ssh-deploy) from 5.1.0 to 5.1.1.
- [Release notes](https://github.com/easingthemes/ssh-deploy/releases)
- [Changelog](https://github.com/easingthemes/ssh-deploy/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/easingthemes/ssh-deploy/compare/ece05a22752e524363164bfb2f69a5ba4f8ded0d...a1aa0b6cf96ce2406eef90faa35007a4a7bf0ac0)

---
updated-dependencies:
- dependency-name: easingthemes/ssh-deploy
  dependency-version: 5.1.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump peaceiris/actions-hugo

Bumps [peaceiris/actions-hugo](https://github.com/peaceiris/actions-hugo) from 6e295a6a0c9087bf374299e9d67f9d2edab9f18f to 75d2e84710de30f6ff7268e08f310b60ef14033f.
- [Release notes](https://github.com/peaceiris/actions-hugo/releases)
- [Changelog](https://github.com/peaceiris/actions-hugo/blob/main/CHANGELOG.md)
- [Commits](https://github.com/peaceiris/actions-hugo/compare/6e295a6a0c9087bf374299e9d67f9d2edab9f18f...75d2e84710de30f6ff7268e08f310b60ef14033f)

---
updated-dependencies:
- dependency-name: peaceiris/actions-hugo
  dependency-version: 75d2e84710de30f6ff7268e08f310b60ef14033f
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

gueldenstone committed:
Run deployment scheduled on :20 of every hour

alinnow committed:
fix day alignment of months that start on a Sunday

alinnow committed:
ci: add preview workflow

dependabot[bot] committed:
Bump actions/checkout from 6.0.1 to 6.0.2

Bumps [actions/checkout](https://github.com/actions/checkout) from 6.0.1 to 6.0.2.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/8e8c483db84b4bee98b60c0593521ed34d9990e8...de0fac2e4500dabe0009e67214ff5f5447ce83dd)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: 6.0.2
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

alinnow committed:
ci: let surge comment on pull requests

Jolg42 committed:
input for url + sync German and English versions
